samples/config.sh

```diff
@@ -0,0 +1,10 @@
+API_KEY="<api-key>"  #Api key value
+RHEL_URL="https://access.cdn.redhat.com/content/origin/files/sha256/4e/4e5c7cccb5ea7622640c97725912222458af743cf994e48f10239734c6a2ee65/rhel-8.3-ppc64le-kvm.qcow2" #Rhel url
```

Comment:
We should take this as input and avoid putting this URL here. This is a stale URL and will be available once the user successfully logs in with subscription id.

Reply:
Removed the URL from here and added comment to update it